### ISSUE TITLE
Not fouad response on create or update pipeline

### ISSUE THREAD

**Issue by hamcher** [jpetrucciani/hubspot3#83]
use plural `pipelines` instead of `pipeline` in the url of create and update methods
according to the description of hubspot

https://api.hubapi.com/crm-pipelines/v1/pipelines/deals?hapikey=demo

```
    def create(self, object_type, data=None, **options):
        data = data or {}
        return self._call(
            "pipeline/{}".format(object_type), data=data, method="POST", **options
        )

    def update(self, object_type, key, data=None, **options):
        data = data or {}
        return self._call(
            "pipeline/{}/{}".format(object_type, key),
            data=data,
            method="PUT",
            **options
        )
```

**Comment by jpetrucciani** [comment]
Hmm, interesting - I'll see if I can get that updated here soon.

**Comment by jpetrucciani** [comment]
This fix is now live on pip as version [`3.2.42`](https://github.com/jpetrucciani/hubspot3/releases/tag/3.2.42)

Thanks again for bringing this to my attention! 😄
